apple/internal/apple_framework_import.bzl

```diff
@@ -353,6 +357,16 @@ target.
                 [apple_common.Objc, AppleFrameworkImportInfo],
             ],
         ),
+        "data": attr.label_list(
```

Comment:
I don't think you need to explicitly list the `data` attribute, since all rules have the attribute by default

Reply:
You do seem to need it, without this it fails today

Reply:
Weird, it's listed under https://docs.bazel.build/versions/master/be/common-definitions.html#common-attributes 🤷

Reply:
I noticed that too, but even rules_swift manually specifies it https://github.com/bazelbuild/rules_swift/blob/b255732244abc7e264e0b484271c10bd8999acab/swift/internal/attrs.bzl#L22-L31